templates/server/fastcgi_params.erb

```diff
@@ -5,12 +5,13 @@ fastcgi_param  REQUEST_METHOD     $request_method;
 fastcgi_param  CONTENT_TYPE       $content_type;
 fastcgi_param  CONTENT_LENGTH     $content_length;
 
-fastcgi_param  SCRIPT_FILENAME    $request_filename;
```

Comment:
Yeah I think it should be left alone.

I'd like the parameter to be renamed, but I don't think it's worth it...